src/components/about/sections/PersonDisplay.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { Col } from "react-bootstrap";
+import "../index.css";
+// i18n translations might still be loaded by the http backend
+// use react's Suspense
+export default function PersonDisplay(props) {
+  const { t } = useTranslation();
+  let photo;
+  // manual picture setting
```

Comment:
no idea what this comment means, please improve it

Reply:
got rid of this switch case -and also its comment- by moving pictures to peopleOrdering